### ISSUE TITLE
Develop XML CAAML Feeds;

### ISSUE THREAD

**Issue by lftgly** [lftgly/MWAvalancheCenter#10]
Continue development of XML CAAML Feeds;
the current PHP versions refresh and display correct content, 
but apparently do not meet XML feed standards (???);
Try subscribing to GNFAC's CAAML feeds and displaying polygons on a map, color-coded by danger rating;

**Comment by lftgly** [comment]
November 20, 2015, I have the XML files updating with add_action save_post; waiting on Carol Peck to test pushing rating values to national map via HTTP POST using curl;

**Comment by lftgly** [comment]
The XML documents are being created/updated 99% correctly, after working on this November 29 and 30, 2015. Carol Peck tested a couple of them, and they work.
Remaining problems:
1. "pushing" the XML to jhavalanche.org not working after many hours testing on January 2, 3, 5, and also while testing the new runout zone polygons on January 7; almost deserves its own issue. 
complicating matters, staging server did not have HTTP POST protocols enabled; 
After I asked Fongemie to open a tecgh support request on Sunday, 1/03/16, WPENGINE.COM supposedly enabled POST on our staging site, but my code still isn't working;
cURL will not work from within WordPress, apparently; after many hours wasted Saturday, 1/03/16;
wp_post_remote is current approach;
2. add_action(save_post,... runs more than once, repeatedly updating the XML files and attempting to POST the XML to jhavalanche.org; caused by WP running autosave and revision-saving processes in the background; - I think I can solve this today;
3. final issue may be trivial, but the xfile locRef does not exist; the CAAML XML file contains a locRef  xref that should point to the GML of the forecast zone polygon...

**Comment by lftgly** [comment]
RESOLVED: add_action(save_post,... runs more than once, repeatedly updating the XML files and attempting to POST the XML to jhavalanche.org; caused by WP running autosave and revision-saving processes in the background, as well as when a post is moved to TRASH;;

**Comment by lftgly** [comment]
STATUS: The XML documents are being created/updated 99% correctly, after working on this November 29 and 30, 2015. Carol Peck tested a couple of them, and they work.
Remaining problems:
1. "pushing" the XML to jhavalanche.org not working after many hours testing on January 2, 3, 5, and briefly on January 7; 
cURL will not work from within WordPress, apparently; 
wp_post_remote is current approach;
2. Trivial?!?: the xfile locRef does not exist; the CAAML XML file contains a locRef xref that should point to the GML of the forecast zone polygon...

**Comment by lftgly** [comment]
STATUS: The XML documents are being created/updated 99% correctly, 
Remaining problems:
1. "pushing" the XML to jhavalanche.org not working after many hours testing on January 2, 3, 5, and briefly on January 7; 
2. Enhancement: the xfile locRef does not exist; the CAAML XML file contains a locRef xref that should point to the GML of the forecast zone polygon...
3. the post_content could be inserted in the CAAML file, following these examples 
http://www.mtavalanche.com/node/9557
http://www.mtavalanche.com/sites/default/files/xml/Bridgers_Forecast.xml
